minio/api.py

```diff
@@ -44,7 +44,8 @@
 
 # Internal imports
 from . import __title__, __version__
-from .compat import urlsplit, range, urlencode, basestring
+from .compat import (urlsplit, queryencode,
```

Comment:
We won't be doing this for now this PR works as is.. We will take this activity as such an issue comes in future. We don't want to introduce subtle changes like this which can have unexpected consequences.